server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/examples/activeTerminals/RestServiceActiveTerminals.java

```diff
@@ -0,0 +1,73 @@
+package com.generalbytes.batm.server.extensions.extra.examples.activeTerminals;
+
+import com.fasterxml.jackson.databind.util.JSONPObject;
+import com.generalbytes.batm.server.extensions.IApiAccess;
+import com.generalbytes.batm.server.extensions.ITerminal;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.core.Response;
+
+@Path("/terminals")
+public class RestServiceActiveTerminals {
+    private static final Logger log = LoggerFactory.getLogger("batm.master.extensions.activeTerminals.ActiveTerminalsExtension");
+    private static long pingDelay = 1000 * 60 * 5;
+
+    @GET
+    @Path("/")
```

Comment:
`@Path("/")` is redundant, it will be mapped to `@Path("/terminals")` even if you omit it